jvb/src/main/kotlin/org/jitsi/videobridge/cc/allocation/SingleSourceAllocation2.kt

```diff
@@ -245,17 +245,15 @@ internal class SingleSourceAllocation2(
         if (selectedLayers.isEmpty()) {
             val minHeight = activeLayers.map { it.layer.height }.minOrNull() ?: return Layers.noLayers
             selectedLayers = activeLayers.filter { it.layer.height == minHeight }
-
-            // This recognizes the structure used with VP9 (multiple encodings with the same resolution and unknown frame
-            // rate). In this case, we only want the low quality layer.
-            if (selectedLayers.isNotEmpty() && selectedLayers[0].layer.frameRate < 0) {
-                selectedLayers = listOf(selectedLayers[0])
-            }
```

Comment:
Removing this will result in all the VP9 SS layers being in the `selectedLayers`.  Is this indeed what we want?